.babelrc

```diff
@@ -1,22 +1,58 @@
 {
   "plugins": [
-    "transform-decorators-legacy"
+    "@babel/plugin-transform-destructuring",
+    [
+      "@babel/plugin-proposal-decorators",
+      {
+        "legacy": true
+      }
+    ],
+    [
+      "@babel/plugin-proposal-class-properties",
+      {
+        "loose": true,
+      },
+    ],
+    "@babel/plugin-syntax-dynamic-import",
```

Comment:
This can be removed! The majority of `stage-1` presets just before the package was deprecated (https://github.com/babel/babel/blob/master/packages/babel-preset-stage-1/README.md#babelpreset-stage-1) are not  used by this project.